product-service/swagger.yml

```diff
@@ -131,3 +123,23 @@ components:
         - description
         - price
         - count
+    NewProduct:
```

Comment:
You don't have to specify the whole entire schema again without the id property - you can use the allOf keyword. Cf.: https://stackoverflow.com/a/60549355